README.md

```diff
@@ -140,32 +140,6 @@ cd ..
 Check your first project
 ------------------------
 
```

Comment:
I've reverted the removal and touched-up the wording of the sentences that already existed here. Also, rebased to newest master.